frontend/src/components/Avatar.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+import '../css/components/_avatar.scss';
```

Comment:
Since **_avatar.scss** won't be used in any other CSS file from now on, rename it to avatar.scss. Remove the underscore.

Reply:
Did it 🙌